src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,58 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Prop = {
```

Comment:
Should be correct name. There is not only one prop but a lot of
```suggestion
type Props = {
```